Algorithms/is_unique.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# CtCI 6th Edition Problem 1.1
+# Implement an algorithm to determine if a string has all unique characters.
+
+
+def is_unique(string):
+    chars = set(string)
+    for char in chars:
+        if string.count(char) > 1:
```

Comment:
Yyou loop through the set (let's say that is O(n)), and call `string.count(char)` which is probably also O(n). That makes me think your algorithm is O(n^2)